docs/data-model.rst

```diff
@@ -653,7 +653,8 @@ Furthermore,
 For simplicity and algorithmic efficiency, mutations must also:
 
 - be sorted by site ID;
-- when there are multiple mutations per site, parent mutations must occur
+- when there are multiple mutations per site, mutations should be ordered by
+  decreasing time, if known, and parent mutations must occur
```

Comment:
s/decreasing/nonincreasing

Adjacent times can be equal, right? Also in the changelong.

Reply:
https://github.com/tskit-dev/tskit/pull/726/commits/4427cd24a7551634867d4d2656f3938389a3fa0c